requirements/layered.yml

```
@@ -68,6 +68,7 @@ tests:
   packages:
     - "pytest"
     - "pytest-json-report"
+    - "asv>0.6.*"
```

I made a mistake in my previous suggestion:
```suggestion
    - "asv=0.6.*"
```